vendor/vendor.json

```diff
@@ -182,6 +194,12 @@
 			"revisionTime": "2016-08-11T16:24:59Z"
 		},
 		{
+			"checksumSHA1": "qS3eZZfhdDM6haTZp7OeyhMDlV8=",
+			"path": "github.com/aws/aws-sdk-go/service/route53domains",
+			"revision": "f80e7d0182a463dff0c0da6bbed57f21369d4346",
```

Comment:
Cool, thanks for the info.

I don't think anything currently needs it (for this PR anyway) - I was just curious. There have been some changes to the route53domains api in the intervening year, but nothing to do with fetching or updating nameservers so far as I can tell.